src/screens/LocalCommunity/index.js

```diff
@@ -95,6 +98,21 @@ const LocalCommunity = () => {
       });
       return item.location_id;
     });
+
+    if (location?.length === 0) {
+      AnalyticsEventTracking.eventTrack(
+        BetterSocialEventTracking.ONBOARDING_LOCATION_FIRST_SELECTED,
+        val
+      );
+    }
+
+    if (location?.length === 1) {
+      AnalyticsEventTracking.eventTrack(
+        BetterSocialEventTracking.ONBOARDING_LOCATION_SECOND_SELECTED,
+        val
+      );
+    }
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic here seems to be tracking the first and second location selections. However, it's not clear what happens when a third or subsequent location is selected. If those are also important events to track, you might want to add additional conditions for them.
```diff
+    if (location?.length >= 2) {
+      AnalyticsEventTracking.eventTrack(
+        BetterSocialEventTracking.ONBOARDING_LOCATION_NTH_SELECTED,
+        val
+      );
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->